File: RangeRev/Range.py
"""
    Author : Pakshal Jain
    Date : 12/12/2022
    Program : To Display Numbers in Entered Range in Reverse Order
"""

##############################################################################################################################################

#Function To Display Range in Reverse Order
def RangeRev(No,No2) :
    i = 0;

    print("Printing Numbers in Entered Range in Reverse Order");

    #Logic
    for i in range(No2,No - 1,-1) :
        print(i,end = "\t");

#######################################################################################################################################

#Main Function
def main() :
    No = 0;
    No2 = 0;

    print("Jay ganesh...");

    #Taking Input From User
    print("Enter Starting Number");
    No = int(input());

    print("Enter Ending Number");
    No2 = int(input());

    if((No < 0)or(No > No2)) :
        print("Invalid Range");
        return;
        
    #Call To Range Function
    RangeRev(No,No2);

#############################################################################################################################################

#Callback Method To Call Main Function
if __name__ == "__main__" :
    #Call To Main Function
    main();

File: RangeRev/Range.java
/*
    Author : Pakshal Jain
    Date : 11/12/2022
    Program : To Display Entered Range in Reverse Number
*/

//Required Libraries
import java.lang.*;
import java.util.*;

//Class Declaration
class Rev
{
    //Characteristics of Class
    int No = 0;
    int No2 = 0;

    //Constructro of Class
    public Rev(int no,int no2)
    {
        this.No = no;
        this.No2 = no2;
    }

    //Beahaviour of Class
    void Display()
    {
        int i = 0;

        System.out.println("Printing Numbers in Reverse Order");
        for(i = this.No2;i >= this.No;i--)
        {
            System.out.print(i+"\t");
        }
    }
}

//Main Class
class Range
{
    //Main Function
    public static void main(String args[])
    {
        int No = 0;
        int No2 = 0;

        System.out.println("Jay Ganesh...");

        //Creating object of Scanner Class To Take Input From User
        Scanner sobj = new Scanner(System.in);

        //Taking Input From User
        System.out.println("Enter Starting Number");
        No = sobj.nextInt();

        System.out.println("Enter Ending Number");
        No2 = sobj.nextInt();

        if((No < 0)||(No > No2))
        {
            System.out.println("Invalid Range");
            return;
        }
        
        //Creating object of Rev Class
        Rev robj = new Rev(No,No2);
        //Call To Behaviour of Class        
        robj.Display();       
    }
}